Yuneec_SDK_iOS/YNCSDKCamera.mm

```diff
@@ -26,6 +26,41 @@ void receive_camera_result(YNCCameraCompletion completion, Camera::Result result
     }
 }
 
+//MARK: receive color mode result
+void receive_color_mode_result(YNCColorModeCompletion completion, Camera::Result result, Camera::ColorMode colorMode) {
+    if (completion) {
+        NSError *error = nullptr;
+        YNCCameraColorMode tmpColorMode = YNCCameraColorMode::YNCUNKNOWN;
+        if (result != Camera::Result::SUCCESS) {
+            NSString *message = [NSString stringWithFormat:@"%s", Camera::result_str(result)];
+            error = [[NSError alloc] initWithDomain:@"Camera"
+                                               code:(int)result
+                                           userInfo:@{@"message": message}];
+            completion(tmpColorMode, error);
+        }
+        else {
+            switch (colorMode) {
+                case Camera::ColorMode::ENHANCED:
+                    tmpColorMode = YNCENHANCED;
+                    break;
+                case Camera::ColorMode::NEUTRAL:
+                    tmpColorMode = YNCNEUTRAL;
+                    break;
+                case Camera::ColorMode::NIGHT:
+                    tmpColorMode = YNCNIGHT;
+                    break;
+                case Camera::ColorMode::UNPROCESSED:
+                    tmpColorMode = YNCUNPROCESSED;
+                    break;
+                default:
+                    tmpColorMode = YNCUNKNOWN;
+                    break;
+            }
+            completion(tmpColorMode, error);
```

Comment:
Do we need to check if `completion` is not null?

Reply:
Isn't the statement if(completion), checking for null?